inst/templates/ad_admh.R

```diff
@@ -8,7 +8,7 @@ library(admiral.test) # Contains example datasets from the CDISC pilot project
 library(dplyr)
 library(lubridate)
 
-# Load source datasets ----
+# ---- Load source datasets ----
```

Comment:
Why is `ad_admh.R` changed in this PR?

Reply:
Not sure. Don't remember updating this.  Should this be reverted back to previous version?

Reply:
Yes